front-end/components/SettingsPanel/panels/Notify/notify.vue

```diff
@@ -48,6 +67,13 @@ export default {
 .option
     @extend %radio-option
 
+.sub-option
+    @extend %check-box
+
 hr
     @extend %line-break
+
+li
+    list-style-type: none
+    padding-left: 20px
```

Comment:
I personally like `35px` better, what do you think?

Reply:
35px looks better indeed